articles/bayesian-search/index.md

```diff
@@ -0,0 +1,161 @@
+
+This twist shows us that Frank’s chances of scoring in the next game he starts have increased considerably. From the example, we have used the concept of probability on a basic statistical problem. This defines Bayesian statistics.
+
+### Bayesian Optimization
+
+Bayesian optimization can be categorized as a sequential model-based optimization algorithm. This means that it carries out trials iteratively as it tries better hyperparameters. It uses the outcome of previous iterations to decide on the next hyperparameters. Bayesian optimization does not search through the hyperparameter space blindly. This contrasts with grid and random search methods. This form of optimization first constructs a probabilistic model. It then maps hyperparameters to a probability of a score on a function. This is the objective function.
+
+The objective function takes in hyperparameters and gives out a score. The score should either be minimized or maximized. The score produced is a single score and is a real value. Objective functions are costly to compute. It is difficult to attempt every single hyperparameter combination. This is because the objective function cannot be calculated quickly. As such, Bayesian optimization aims to cut down on the number of times we would need to run the function. It calculates the conditional probability of the objective function. This calculation is for a validation set, having a set of hyperparameter values used to train the probabilistic model. Conditional probability is the probability of an event taking place, given that a different event has occurred.
+
+### Steps to apply Bayesian optimization
+
+As we mentioned above, the objective function is difficult to use in computations. As a result, a simper function is considered in place of the objective function. This simpler function is easier to optimize. We call it the surrogate function. It is the true objective function represented in terms of probability. This is as a result of previously-evaluated selected hyperparameters. We shall discuss this function further in the next section.
+
+To carry out Bayesian optimization for hyperparameter search, we follow the steps below:
+
+1. Create a surrogate probability model of the objective function.
+
+2. Find the surrogate model’s best-performing hyperparameters.
+
+3. Utilize the best performing hyperparameters on the true objective function.
+
+4. Update the surrogate probability model. This is done by including new results from the previous step.
+
+5. Repeat the second to fourth step until we achieve a pre-determined completion condition.
+
+We will go through the mentioned steps in detail during an in-depth tutorial in the future. For now, our scope is on the basics of Bayesian optimization.
+
+### Components of Bayesian optimization
+
+There exist two main components of Bayesian optimization:
+
+· Surrogate functions
+
+· Acquisition functions
+
```

Comment:
Please make changes on the list format. These appear to be full stops rather than bullet points